driver_go18_test.go

```diff
@@ -98,6 +106,118 @@ func getQueryProcess(processes []dbProcess, query string) *dbProcess {
 	return nil
 }
 
+func pollQueryProcess(db *sql.DB, dbName, query string, totalTimeout, pollTimeout time.Duration) (*dbProcess, error) {
```

Comment:
`getQueryProcess(db, query, timeout)`. the separate method getQueryProcess does not need.